qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/VerifyLoginSteps.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import org.openmrs.contrib.qaframework.RunTest;
+
+public class VerifyLoginSteps extends Steps {
+
+    @Before(RunTest.HOOK.SELENIUM_DEFAULT_LOGIN)
+    public void visitHomePage() {
+        initiateWithLogin();
+    }
+    
+    @After(RunTest.HOOK.SELENIUM_DEFAULT_LOGIN)
+    public void destroy() {
+        quit();
+    }
+
+    @And("user verifies modules available on home page after login as admin")
+    public void verifyAdminModulesAvailableOnHomePage() {
+        goToLoginPage().loginAsAdmin();
+        assertTrue(homePage.isFindAPatientAppPresent());
+        assertTrue(homePage.isActiveVisitsAppPresent());
```

Comment:
I see none buttons here: https://qa-refapp.openmrs.org/openmrs/referenceapplication/home.page
Is there any reason why this guy can only see eight?

Reply:
We didnot consider report app since it doesnt have much work , we also didnot include it on home page. Unless you are okay me adding it here.